src/python/platforms/fuchsia/util/fuzzer.py

```diff
@@ -73,9 +73,11 @@ def _is_example(cls, pkg, tgt):
     """ Returns whether or not a given pkg/tgt pair is an example fuzzer.
     (Helper function to prevent us from wasting cycles on example fuzzers in
     production). """
-    # We want to remove the noop-fuzzer regardless of the extension.
+    # Strip any sanitizer extensions
     tgt = os.path.splitext(tgt)[0]
-    return pkg == 'example_fuzzers' or tgt == 'noop-fuzzer'
+    return (pkg == 'example_fuzzers' and tgt != 'oom_fuzzer') \
```

Comment:
nit: we generally don't use \ for line continuations. Use parentheses instead.

i.e.

```python
return ((pkg == 'example_fuzzers' and tgt != 'oom_fuzzer')
        or (pkg == 'zircon_fuzzers' and tgt == 'noop-fuzzer')...
```

Reply:
Done!